package.nls.json

```diff
@@ -180,6 +180,7 @@
     "AWS.command.ecr.createRepository": "Create Repository...",
     "AWS.command.ecr.deleteRepository": "Delete Repository...",
     "AWS.command.ecr.deleteTag": "Delete Tag...",
+    "AWS.command.ecs.viewDocumentation": "View ECS Exec Documentation",
```

Comment:
Why not use the generic `AWS.generic.viewDocs` ? 

Please always consider precedent when making changes. There is a "CDK" case, but it is alone, thus it's not a precedent (and really it should also use `AWS.generic.viewDocs`).

We want to move towards generalizing

Reply:
Agreed - context should be enough to make it clear what "View Docs" is going to take you to.